### ISSUE TITLE
Validation loss vs. metrics?

### ISSUE THREAD

**Issue by cy94** [UMass-Foundation-Model/3D-LLM#66]
Dear authors, @evelinehong,

Thanks for the interesting paper and for releasing the code and models :) I was able to reproduce the ScanQA results on the validation set. I also added computation of the validation loss, similar to the training step by calling `forward` in addition to `predict_answers`, in `VQATask.valid_step`. However I noticed while the validation loss goes up, the validation metrics also go up.
![image](https://github.com/UMass-Foundation-Model/3D-LLM/assets/5152802/7c22c2c7-12f8-47af-804c-76408085301e)

![image](https://github.com/UMass-Foundation-Model/3D-LLM/assets/5152802/be8bee26-25ca-4f87-b1a4-421d2061fefd)

![image](https://github.com/UMass-Foundation-Model/3D-LLM/assets/5152802/83908fb9-0cd5-4dad-95ac-5bad39205c12)

Did you notice something similar to this while training, or do you have any suggestions as to why this could happen? I would expect validation loss and metrics to be correlated, even if the val loss is not from autoregressive generation. 

Best,
Chandan
